dashboards/Au2mator/readme.md

```diff
@@ -0,0 +1,44 @@
+# Au2mator Dashboard
+
+In the first half, we use SQL and PowerShell tiles to present Performance and Health Counters for the au2mator- Self Service Portal and a Web Tile to directly open the Self Service Portal in a new Tab.
+
+### Total Requests
+The SQL Donut Tile is displaying the actual State of all Requests and their State. Here we are using the “Timeframe-Option.”
+
+### Heavy Users
+Then we used the PowerShell Donut Tile cause the au2mator DB stored the SamAccountName, and with PowerShell, we get the Displayneme from Active Directory.
+
+### Total Services
+Just a Count on the number of our au2mator Services.
+
+### Top 8 Service Usage
+Also, a SQL Donut Tile to present the Top 8 Service by usage
+
+### Start au2mator
+A Web Tile to start the au2mator – Self Service Portal in a new Tab
+
+![image](https://user-images.githubusercontent.com/37934234/129005052-d158c62f-dea7-478a-b15b-e6fccf84a195.png)
```

Comment:
This should be switched to the permalink for the image in the images folder (see slack)